### PR TITLE
Drop Python 2

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

**Comment by cschanaj** [comment]
cc @Hainish @zoracon Python 2.7 will retire in days and no longer be maintained (https://pythonclock.org/) . would you take a look at this PR?

File: Dockerfile [hunk 1]
```diff
@@ -18,15 +17,11 @@ RUN apt-get update && \
     libssl-dev \
     ca-certificates \
     miredo \
-    tor && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/* \
-    /tmp/* \
-    /var/tmp/*
```
Comment (Hainish) [inline Dockerfile:25]: Why remove the `apt-get clean` and `rm -rf` statements?

Comment (J0WI) [reply Dockerfile:25]: They are superfluous

**Comment by Hainish** [comment]
Thank you!

**Pull request merged**